build/lib/packager.js

```diff
@@ -275,10 +275,12 @@ class Packager {
 			input: `${tmpBundleDir}/Resources/ti.main.js`,
 			plugins: [
 				resolve(),
-				commonjs(),
+				commonjs({
+					ignore: [ '/semantic.colors.json' ]
+				}),
 				babel(babelOptions)
 			],
-			external: [ './app', 'com.appcelerator.aca' ]
+			external: [ './app', 'com.appcelerator.aca', '/semantic.colors.json' ]
```

Comment:
@ewanharris I am getting error, if I run titanium SDK using Xcode.
`[ERROR] Script Error {
    column = 46;
    line = 5;
    message = "Couldn't find module: ../../../semantic.colors.json for architecture: x86_64";`

Reply:
I forgot to copy the rollup options to the `scons-xcode-project-build` script. Will fix

Reply:
Fixed in 8e732a37a3c1161f7ade223f85fc13734fb5272d